src/database/sqlite/adapter.ts

```diff
@@ -0,0 +1,101 @@
+import { eq, and } from "drizzle-orm"
+import {
+  integer,
+  sqliteTable as defaultSqliteTableFn,
+  text,
+  BaseSQLiteDatabase,
+  SQLiteTableFn,
+} from "drizzle-orm/sqlite-core"
+import { Asset } from '../../assets'
+
+export function createTables(sqliteTable: SQLiteTableFn) {
+  const sources = sqliteTable("sources", {
+    id: integer('id').primaryKey({autoIncrement: true}).notNull(),
+    src: text('src').notNull(),
+    type: text('type'),
+    videoId: integer('video_id').references(() => videos.id).notNull(),
+  })
+
+  const videos = sqliteTable("videos", {
+    id: integer('id').primaryKey({autoIncrement: true}).notNull(),
+    status: text('status', {enum: [
+      'sourced',
+      'pending',
+      'uploading',
+      'processing',
+      'ready',
+      'error',
+    ]}).notNull(),
+    originalFilePath: text('original_file_path').notNull(),
+    provider: text('provider', {enum: [ 'mux',  'vercel-blob', 'backblaze', 'amazon-s3']}).notNull(),
+    providerMetadata: text('provider_metadata', {mode: 'json'}).notNull(),
+    poster: text('poster'),
+    blurDataURL: text('blur_data_url'),
+    size: integer('size'),
+    error: text('error'),
+    externalIds: text('external_ids', {mode: 'json'}),
```

Comment:
`externalIds` can be removed here. these are stored in providerMetadata in the new schema

Reply:
Great! I just put it in there because I saw some comments about backwards compatibility